testsuite/features/support/system_monitoring.rb

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 SUSE LLC.
+# Licensed under the terms of the MIT license.
+
```

Comment:
This is not `system_monitoring`. These are supplementary functions to be used by Quality Intelligence exclusively. I would integrate them to `QualityIntelligence` class and use them directly there.

Reply:
I really prefer to keep QualityIntelligence simpler and less coupled with the rest of our test framework code.

Reply:
ok